packages/client/lib/__tests__/MetadataClient.spec.ts

```diff
@@ -38,8 +38,8 @@ describe('Metadataclient with IdentiProof Issuer should', () => {
 
     const INITIATE_URI =
       'openid-initiate-issuance://?issuer=https%3A%2F%2Fissuer.research.identiproof.io&credential_type=OpenBadgeCredential&pre-authorized_code=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJhOTUyZjUxNi1jYWVmLTQ4YjMtODIxYy00OTRkYzgyNjljZjAiLCJwcmUtYXV0aG9yaXplZCI6dHJ1ZX0.YE5DlalcLC2ChGEg47CQDaN1gTxbaQqSclIVqsSAUHE&user_pin_required=false';
-    const initiation = IssuanceInitiation.fromURI(INITIATE_URI);
-    const metadata = await MetadataClient.retrieveAllMetadataFromInitiation(initiation);
+    const initiation = new IssuanceInitiation(INITIATE_URI).issuanceInitiationWithBaseUrl;
```

Comment:
Please do not change builder like patterns to direct constructor patterns. Doesn't make sense

Reply:
https://github.com/Sphereon-Opensource/OpenID4VCI/pull/31#discussion_r1146021508

Reverted the changes.